docs/rules/role-supports-aria-props.md

```diff
@@ -0,0 +1,74 @@
+# Role Supports ARIA Props
```

Comment:
This was adapted from https://github.com/github/erblint-github/blob/main/docs/rules/accessibility/no-aria-label-misuse-counter.md